File: js/programmers/level_0/짝수는 싫어요.js
// Lv. 0 짝수는 싫어요

/**
 * @param {number} n
 * @returns {number[]}
 */

const solution = (n) => {
    const answer = [];
    let i = 1;

    while (i <= n) {
        answer.push(i);
        i += 2;
    }

    return answer;
};

File: java/programmers/level_0/src/짝수는_싫어요.java
import java.util.Arrays;

public class 짝수는_싫어요 {

  public static int[] solution(int n) {

    int[] answer = new int[(n + 1) / 2];

    for (int i = 0; i < answer.length; i++) {
      answer[i] = i * 2 + 1;
    }
    return answer;
  }

  public static void main(String[] args) {
    System.out.println(Arrays.toString(solution(10)));
  }
}
